### DISCUSSION TITLE
Cursor speed [Moved to Issue #409]

### DISCUSSION THREAD

**Discussion by Barondane** [noah-nuebling/mac-mouse-fix#408] [Other]
I suggest increasing the price of the app while adding more functionality and device support.

I tried the app for few minutes. I wanted to replace Logi options. There are nice features in the app that I did not even use. Main issue for me: Cursor speed was not manageable via the app. Mac mouse setting is slow to my use. It's possible that I missed this feature while testing the app. If so, please let me know. 

I am very supportive of this app. Thanks for making and maintaining it. I will be using it as soon as speed is controllable.

**Comment by krackers** [comment]
By cursor speed are you referring to sensitivity or acceleration? See https://plentycom.jp/en/cursorsense for an explanation of the difference.

**Comment by Barondane** [comment]
I would say both

**Reply by krackers** [reply]
Might want to try linearmouse, as far as I'm aware MMF doesn't support this. Although the code for it exists in abandoned branch and is actually the same code that linearmouse uses. 

It should not be too hard to integrate though, since it's really just a few IOReg calls, not sure if the dev wants to expand the scope of the app though

**Reply by Barondane** [reply]
Very helpful, thank you!

**Comment by noah-nuebling** [comment]
Hey guys, I've moved the topic to Issue #409, so I can keep a better overview of all the bugs and feature requests!
